static/app/views/dashboardsV2/widgetCard/index.tsx

```diff
@@ -223,6 +223,7 @@ const StyledPanel = styled(Panel, {
   min-height: 96px;
   display: flex;
   flex-direction: column;
+  overflow: hidden;
```

Comment:
![image](https://user-images.githubusercontent.com/29228205/154967552-6141a2c4-5684-4123-b776-e7831f477f3d.png)

happens when the widget is loading

Reply:
oh noo :( 

good catch @narsaynorath thank you! sure I'll remove the overflow and we can maybe address this in a separate PR